css/style.css

```diff
@@ -231,14 +230,128 @@ width: 600px;
     margin-bottom: 20px;
 }
 
-.rent__text + .rent__text {
+.rent__text+.rent__text {
     margin-bottom: 20px;
 }
 
-.rent__item{
+.rent__item {
     border: 1px solid #332208;
     width: 250px;
- padding: 0 30px 10px;
+    padding: 0 35px 10px;
     text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+
+
+}
+
+.rent__item-num {
+    font-family: "Fira Sans", sans-serif;
+    font-weight: 500;
+    font-size: 96px;
+    line-height: 115px;
+    opacity: 0.05;
+}
+
+.rent__item-title {
+    font-family: "Fira Sans", sans-serif;
+    font-size: 18px;
+    line-height: 22px;
+    margin-bottom: auto;
+}
+

```

Comment:
Please try to write styles for different sections in different css files. It's a good practice https://developer.mozilla.org/en-US/docs/Web/CSS/@import